test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
+
```

Comment:
[NITS] This line is redundant

Reply:
Could you explain what does [NITS] stand for ? And what line of the above lines is redundant ? (1,2,3,4)